src/symmbasis.jl

```diff
@@ -237,6 +237,27 @@ function genmul!(C, xA::Transpose{<:Any,<:AbstractSparseMatrixCSC}, B, mulop)
    return C
 end
 
+#dispatching for SVectors
```

Comment:
I thought before that the standard implementation will "just work" what did you have to change?Is it the line 
```
C[rv[j], k] += mop * ones(SVector{length(C[1]),eltype(mop)})
```
???

Reply:
When calculating the derivative according to parameters of a linear model with one property we expect a single vector for return. When we do this for multiple properties we expect several vectors, however these are the same since it's the derivative of a linear model w.r.t parameters. The difference in the ```genmul!``` is that the one I added returns a vector of SVectors rather than a regular vector.
What I changed was for ```genmul!``` to return the same result N times where N is the number of properties.

Reply:
hm ... so that almost sounds like another argument for switching to matrices? we should still go ahead and merge this but need a serious conversation about this choice.